demo/orfs-gcd-timing/README.md

```
@@ -1,221 +1,336 @@
-# Check for violations
-report_checks -path_delay max -slack_max 0.0
-```
+**Clock Period vs Fmax**:
+- `period_min` in reports shows minimum achievable clock period
+- `fmax` = 1 / period_min
+
+### Reading Timing Paths
 
-**Expected Violation (Verified):**
+Example from report:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)